.github/workflows/macos.yml

```diff
@@ -71,9 +84,36 @@ jobs:
 
       - run: make
 
+      # Launchable requires Python and Java
+      # https://www.launchableinc.com/docs/resources/cli-reference/
+      - name: Set up Python
+        uses: actions/setup-python@871daa956ca9ea99f3c3e30acb424b7960676734 # v5.0.0
+        with:
+          python-version: "3.10"
+      - name: Set up JDK 17
+        uses: actions/setup-java@7a445ee88d4e23b52c33fdc7601e40278616c7f8 # v4.0.0
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Setup Launchable
+        id: setup_launchable
+        run: |
+          pip install launchable
+          launchable verify
+          launchable record build --name ${GITHUB_PR_HEAD_SHA}
+          : # To avoid file name conflicts, we use a random number here as a ID.
+          random_number=$RANDOM
+          echo "random_number=$random_number" >> $GITHUB_OUTPUT
+          echo "TEST_WITH_LAUNCHABLE=--launchable-test-reports=$random_number.json" >> "$GITHUB_ENV"
+        if: ${{ github.repository == 'ruby/ruby' && (matrix.test_task == 'check' || matrix.test_task == 'test-all') }}
+
+      - name: Set repeat-count option for make ${{ matrix.test_task }}
+        run: echo "TEST_WITH_LAUNCHABLE=$TEST_WITH_LAUNCHABLE --repeat-count=2" >> "$GITHUB_ENV"
```

Comment:
Why did you add the `--repeat-count=2` here? Why did you add it only if the `matrix.test_task == 'test-all'`?  The option doesn't work if the `matrix.test_task == 'check'`?

Reply:
Perhaps, you just wanted to keep the compatibility from the current master's behavior, because you removed the `TESTS=--repeat-count=2` in the `test_task: test-all TESTS=--repeat-count=2` in this PR.

Reply:
Yes that's right.

Reply:
All right!